### ISSUE TITLE
Small doc fixes

### ISSUE THREAD

**Issue by fionaEyoung** [fionaEyoung/pytorch#1]
### 📚 The doc issue

Typo in [`utils.data.get_worker_info()`](https://github.com/fionaEyoung/pytorch/blob/56879f64a8d72260984d999a6a87fc4394855578/torch/utils/data/_utils/worker.py#L124) 
`# **sharded** dataset, or use ``seed`` to seed other libraries used in dataset`

Link to docs: https://docs.pytorch.org/docs/stable/data.html#torch.utils.data.get_worker_info

### Suggest a potential alternative/fix

sharded→ shared
